ReactiveCocoa.xcodeproj/xcshareddata/xcschemes/ReactiveCocoa-Mac.xcscheme

```diff
@@ -3,8 +3,8 @@
    LastUpgradeVersion = "0700"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "YES"
-      buildImplicitDependencies = "YES">
+      parallelizeBuildables = "NO"
```

Comment:
That explains `buildImplicitDependencies` (which I agree with, I'm :+1: for explicitness), but why `parallelizeBuildables`?